Data/uomReferenceData-starter-set.ttl

```diff
@@ -0,0 +1,2650 @@
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_absorbed_dose> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/semanticarts/ns/ontology/gist/Aspect> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_absorbed_dose> <http://www.w3.org/2000/01/rdf-schema#comment> "Note that the absorbed dose is not a good indicator of the likely biological effect. 1 Gy of alpha radiation would be much more biologically damaging than 1 Gy of photon radiation for example. Appropriate weighting factors can be applied reflecting the different relative biological effects to find the equivalent dose. The risk of stoctic effects due to radiation exposure can be quantified using the effective dose, which is a weighted average of the equivalent dose to each organ depending upon its radiosensitivity. When ionising radiation is used to treat cancer, the doctor will usually prescribe the radiotherapy treatment in Gy. When risk from ionising radiation is being discussed, a related unit, the Sievert is used." .
```

Comment:
Is "stoctic" here a typo or just a word I'm not familiar with?

Reply:
changed "stoctic effects" to "toxic effects"